simularium_models_util/common/readdy_util.py

```diff
@@ -1309,7 +1324,7 @@ def analyze_frame_get_chain_of_types(
             return result
         result.append(n_id)
 
-        if chain_length == 1:
+        if chain_length == 0:
```

Comment:
this change makes the recursive function end early, instead add a check before line 1330 to make sure `next_neighbor_index` is not `None`